### PR TITLE
Zenodo

### PR DESCRIPTION
**Pull request by Nosferican**
- Added Zenodo shield
- Use OSI standardized MIT license so it is machine detectable

### CODE REVIEW

**Comment by Nosferican** [comment]
@nalimilan if that looks good I will open the other PR for StatsBase and StatsModels.

File: README.md [hunk 1]
```diff
@@ -15,3 +15,6 @@
 
 [coveralls-img]: https://coveralls.io/repos/github/JuliaStats/GLM.jl/badge.svg?branch=master
 [coveralls-url]: https://coveralls.io/github/JuliaStats/GLM.jl?branch=master
+
+[DOI-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.3376014.svg
+[DOI-url]: https://doi.org/10.5281/zenodo.3376014
```
Comment (nalimilan) [inline README.md]:
```suggestion
[DOI-url]: https://doi.org/10.5281/zenodo.3376013
```

File: README.md [hunk 2]
```diff
@@ -15,3 +15,6 @@
 
 [coveralls-img]: https://coveralls.io/repos/github/JuliaStats/GLM.jl/badge.svg?branch=master
 [coveralls-url]: https://coveralls.io/github/JuliaStats/GLM.jl?branch=master
+
+[DOI-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.3376014.svg
```
Comment (nalimilan) [inline README.md]: Better use the DOI referring to all version of GLM: we don't want to update the README for every new release. Users can find the DOI for the version they are using on Zenodo.

```suggestion
[DOI-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.3376013.svg
```

File: README.md (hunk 1 above)
Comment (nalimilan) [reply README.md]: This one also applies.

**Comment by nalimilan** [comment]
Can you move the LICENSE changes to another PR? I'd rather wait until a solution is decided in Example.jl.

**Comment by Nosferican** [comment]
Yes, I can work on this in the evening.

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**